tdp/core/variables.py

```diff
@@ -61,110 +64,45 @@ def __init__(self, content):
         """
         self._content = content
 
+    def copy(self):
+        return self._content.copy()
+
+    def merge(self, mapping):
```

Comment:
This function return something ? Can you do something like `Variables(...).merge(...).merge(...)` to perform multiple merge or it is useless ?

Reply:
It would need to return `self`, I think this construct would be weird on a dict, I'd rather call sequentially multiple merge operation.